src/components/SingleArticle.jsx

```diff
@@ -20,20 +21,14 @@ const SingleArticle = () => {
     if (isLoading) {
         return <p>Loading ...</p>
     }
-
-
     return (
         <div>
             <section className="ArticleDetails">
             <h2 className='ArticleTitle'>{articleDisplay.title}</h2>
             <p className='ArticleAuthor'>Author: {articleDisplay.author}</p>
             <p className='ArticleDate'>{articleDisplay.created_at} </p>
             <p className='ArticleBody'>{articleDisplay.body}</p>
-            <p className='ArticleVotes'>Votes: {articleDisplay.votes}</p>
-            </section>
-            <section className="VoteSection">
-            <button className="UpVoteButton"></button>
-            <button className="DownVoteButton"></button>
+            <ArticleVotes article_id={articleDisplay.article_id} articleVotes={articleDisplay.votes}/>
```

Comment:
This is a nice refactor, moving the 2 buttons into their own component - well done!